### DISCUSSION TITLE
Docker开启补偿机制模式

### DISCUSSION THREAD

**Discussion by tianniao886** [Guovin/iptv-api#729] [Ideas]
新拉取Docker版，config中没有open_supply = True，自行添加后会出现错误。能否也添加个补偿机制啊？感谢作者的付出！！
![13](https://github.com/user-attachments/assets/af002f53-81b2-4a92-a549-71dd4feced18)

**Comment by Guovin** [comment]
完整版的docker镜像还没有上传更新，旧版本是没有这个配置的，等更新后再拉取运行

**Comment by Guovin** [comment]
已上传，可拉取最新镜像了
